packages/dashboard/src/lib/framework/extension-api/types/detail-forms.ts

```diff
@@ -58,6 +58,10 @@ export interface DashboardDetailFormDisplayComponent {
     component: DataDisplayComponent;
 }
 
+/**
+ * @description
+ * Allows you to extend existing detail forms with custom GraphQL queries, input components, and display components.
```

Comment:
maybe add "(e.g. on the product detail or customer detail pages)" just to be explicit about what we mean with "detail form"